site/diagrams/sequenceDiagrams/command_control_concept_http.txt

```diff
@@ -0,0 +1,18 @@
+participant Device
+participant HTTP Adapter
+participant AMQP 1.0 Network
+participant Application
+
+Device->HTTP Adapter: 1. send event (hono-ttd=30)
+activate HTTP Adapter
+HTTP Adapter->Application: 2. connected notification (ttd=30)
+activate Application
+Application->HTTP Adapter: 3. transfer("control/TENANT/4711","light","ON")
+HTTP Adapter->Device: 4. event response("light","ON",${reqId})
```

Comment:
according to my comment above:
`HTTP response ("202", "hono-cmd=light, hono-cmd-req-id=${reqId})", "ON")`